advanced-examples/system-performance/stressng-pcp/config.yaml

```diff
@@ -0,0 +1,10 @@
+log:
+  level: debug
+logged_outputs:
+  error:
+    level: debug
+deployers:
+  image:
+    deployer_name: podman
+    deployment:
+      imagePullPolicy: IfNotPresent
```

Comment:
I thought they might be required, but testing proves otherwise. As far as the default, that is probably container platform dependent.

Reply:
Even if this isn't required, it provides a good starting point for users.

Reply:
But, @jaredoconnell, do we need to provide this "starting point" in _every_ example??  That makes it look like the file and this key are _required_, when they are actually the default.

Reply:
If they are present and the user wants to edit them, they can just edit them. If they are left out to use the default value, the user would instead need to look up the config keys to use them, and they would need to look up what the default value is. It's more convenient.

Reply:
By that logic, every example should provide every key in every file.  And, if we add keys or change keys, then we have to find and edit every one of those files.  And, it means that the examples are bigger than they need to be and provide details which distract and detract from the purpose of the example.

It would be better to provide only what we need to in order to demonstrate the feature that the example is supposed to highlight -- this is better for the person consuming the example and it's better for the people maintaining it.

If someone needs a "starting place", we can provide template or an example for that -- then there is one place where all that stuff is, and only one place which has to be kept up.

In the case cited above, it's unlikely that anyone is ever going to need or want to override the default pull policy.  And, most likely, anyone running this example will be satisfied with the default configuration, so the whole file is superfluous.  As long as we can provide reasonable defaults, we can save ourselves from having to maintain files like this one.  So, we win by delivering less, here.